### DISCUSSION TITLE
set datetime with mqqtt

### DISCUSSION THREAD

**Discussion by sylvaing** [john30/ebusd#841] [Q&A] [answered]
Hi all, and thanks @john30 for ebus apadpter v3 and ebusd !

I installed my adapter and play with this and home assistant, and it works perfectly.

I try to write a datetime of vacation start of my controller. HA discovery don’t support datetime, so I try to make it manually, but it doesn’t work by mqtt,
All my config since ok with filter-rules and access level. I can change some standard value provided by discovery.

When I use command line it works

```
bash-5.1# ebusctl r VacationStartTime
13.02.2023 11:10

bash-5.1# ebusctl w -c e7f_2 VacationStartTime '15.02.2023 12:20'
done

bash-5.1# ebusctl r VacationStartTime
15.02.2023 12:20
```

but When I try to send a payload with mqtt I have error on ebusd:
```
2023-02-16 12:18:15.268 [bus error] prepare message part 0: ERR: invalid numeric argument
2023-02-16 12:18:15.268 [mqtt error] write e7f_2 VacationStartTime: ERR: invalid numeric argument
```

topic:
```
ebusd/e7f_2/VacationStartTime/set
```
payload:
```
12.02.2023 10:00
```

I think the payload is incorrect, but what is the correct payload format for a datetime trough MQTT ?

**Comment by Dinth** [comment]
Not sure if this helps but...
![D56763A4-BCA2-480C-A1EF-25297C65372E](https://user-images.githubusercontent.com/313674/219385471-ade5392b-5ff3-40fa-a65a-25ff00f2e396.jpeg)

**Reply by sylvaing** [reply]
Thank you for your screenshot, but it's datetime, not just a date.
But I see that for a "date", we can set just a "string" of date, and seems that not working for a datetime.

**Comment by sylvaing** [comment, marked as answer]
After a good night, I restested this...
And now it works like charm.. I think that a introduce wrog character perhaps.. but now it's ok:
<img width="1274" alt="Screenshot 2023-02-17 at 10 08 07" src="https://user-images.githubusercontent.com/71899/219601890-9d2f89bb-7e62-4937-b7d6-33a3539526e9.png">

thank you @Dinth
